src/Lachain.Benchmark/BlockchainBenchmark.cs

```diff
@@ -111,8 +111,9 @@ public void Start(RunOptions options)
             Console.WriteLine("-------------------------------");
 
             Logger.LogInformation($"Setting chainId");
-            var chainId = _configManager.GetConfig<NetworkConfig>("network")?.ChainId;
-            TransactionUtils.SetChainId((int)chainId!);
+            var chainId = _configManager.GetConfig<NetworkConfig>("network")?.ChainId ?? 42;
```

Comment:
Should we hardcode this "42" ? Is it better to throw an exception ?

Reply:
it doesn't matter here really,  I'll added it here to not force config editing for benchmark.